github/templates/README.md

```diff
@@ -0,0 +1,79 @@
+PROJECT NAME
+============
+
+**Build Badges**
+
+Short project description. You can use the client's pitch. No more than ten lines.
+
+**Table of Contents**
+
+* [Setup](#setup)
+* [Development](#development)
+* [Deployment](#deployment)
+* [More Sections](#more-sections)
+* [Contribution Guidelines](#contribution-guidelines)
+* [Resources](#resources)
+* [Contacts](#contacts)
+
+## Setup
+
+A set of instructions to install the project. It can start with a list of dependencies, like databases, and finish with the instructions. For instance:
+
+First clone the repository to your file system:
+
+```
+git clone git@github.com:subvisual/guides.git
+```
+
+After, run the setup script:
+
+```
+bin/setup
+```
+
+To finish, open the file `.env` and replace `DB_USERNAME` and `DB_PASSWORD` with the correct username and password for the Postgres user.
```

Comment:
It's just an example, it's not indicating anything. It's so that people have an idea of what to write in this section. Almost all projects require setting up the database's username and password.